lib/ebpf/BUILD

```diff
@@ -0,0 +1,10 @@
+package(default_visibility = ["//lib:__subpackages__"])
+
```

Comment:
Since there can be 2 types of ebpf code, one which runs in kernel mode (restricted C code) and another which is userspace ebpf code, it is better to name the directory/file accordingly with _kern or _user suffix.

Reply:
Discussed over slack. The ebpf/ directory only contains .c code that is compiled into eBPF to run in restricted mode in the kernel (eg, the _kern code). Management code will be C++ in the parent directory (eg, the _user code_).

I have added a README indicating the purpose of the ebpf/ directory. Given the difference in language and build rules, the different directories, and the README, we were both ok not adding further suffixes.